demos/twa-notification-delegation/src/main/AndroidManifest.xml

```diff
@@ -14,6 +14,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.google.androidbrowserhelper.demos.twa_notification_delegation">
 
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```

Comment:
Just to check, what happens if you try to install this on a pre-T device? I can never remember how AndroidManifest.xml works with backwards compatibility.